MembraneRTC/src/main/java/org/membraneframework/rtc/media/SimulcastVideoEncoderFactoryWrapper.kt

```diff
@@ -0,0 +1,208 @@
+package org.membraneframework.rtc.media
```

Comment:
Did we copied it? If yes, we should include original license with copyrights at the beginning of the file